src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/mapper/OffenderMapper.java

```diff
@@ -7,21 +7,9 @@
 import java.util.function.BiFunction;
 import java.util.stream.Collectors;
 import org.springframework.stereotype.Component;
-import uk.gov.justice.probation.courtcaseservice.restclient.communityapi.model.CommunityApiConvictionResponse;
-import uk.gov.justice.probation.courtcaseservice.restclient.communityapi.model.CommunityApiConvictionsResponse;
-import uk.gov.justice.probation.courtcaseservice.restclient.communityapi.model.CommunityApiOffenderManager;
-import uk.gov.justice.probation.courtcaseservice.restclient.communityapi.model.CommunityApiOffenderResponse;
-import uk.gov.justice.probation.courtcaseservice.restclient.communityapi.model.CommunityApiRequirementResponse;
-import uk.gov.justice.probation.courtcaseservice.restclient.communityapi.model.CommunityApiRequirementsResponse;
-import uk.gov.justice.probation.courtcaseservice.restclient.communityapi.model.CommunityApiSentence;
-import uk.gov.justice.probation.courtcaseservice.restclient.communityapi.model.CommunityApiUnpaidWork;
-import uk.gov.justice.probation.courtcaseservice.service.model.Conviction;
-import uk.gov.justice.probation.courtcaseservice.service.model.Offence;
-import uk.gov.justice.probation.courtcaseservice.service.model.OffenderManager;
-import uk.gov.justice.probation.courtcaseservice.service.model.ProbationRecord;
-import uk.gov.justice.probation.courtcaseservice.service.model.Requirement;
-import uk.gov.justice.probation.courtcaseservice.service.model.Sentence;
-import uk.gov.justice.probation.courtcaseservice.service.model.UnpaidWork;
+import uk.gov.justice.probation.courtcaseservice.controller.model.CurrentOrderHeaderResponse;
+import uk.gov.justice.probation.courtcaseservice.restclient.communityapi.model.*;
+import uk.gov.justice.probation.courtcaseservice.service.model.*;
```

Comment:
There was a review comment on Community API from DPS team that they had stopped wildcard imports. Perhaps we could follow suit.

Reply:
Yes I agree, I would prefer to not use wildcard imports by choice but intellij sneaked it in for me. I'll change 🙂